templates/CRM/Contribute/Form/AdditionalPayment.tpl

```diff
@@ -100,7 +100,7 @@
             </td>
           </tr>
           <tr class="crm-payment-form-block-payment_instrument_id">
-            <td class="label">{$form.payment_instrument_id.label}</td>
+            <td class="label">{$form.payment_instrument_id.label} <span class="crm-marker" title="{ts}This field is required.{/ts}">*</span></td>
```

Comment:
Because the same form is used to pay as a credit card using payment processor, in those case this field should not be required.